model/installation_request.go

```diff
@@ -472,6 +477,25 @@ func (p *PatchInstallationRequest) addMissingIngressSourceRanges(installation *I
 	return allowiplistRange
 }
 
+func (p *PatchInstallationRequest) filterInvalidIngressSourceRanges(ips string) string {
+	var validIPs []string
+
+	allowedIPRanges := strings.Split(ips, ",")
+
+	for _, ip := range allowedIPRanges {
+		if IsIPRangeValid(ip) {
+			validIPs = append(validIPs, ip)
+		} else if !IsIPRangeValid(ip) {
+			logrus.Info("Invalid IP range will not be appended: ", ip)
```

Comment:
I believe it would be better to return an error here. Currently, our patching logic enforces a completely valid request or errors so we should probably stick to that and avoid silent failures.